etc/travis/test-images-dockerhub.sh

```diff
@@ -0,0 +1,11 @@
+# fail fast
```

Comment:
Given how we're calling these scripts from `.travis.yml`, do we need to add [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top of these scripts?

```
#!/bin/bash
...
```